docs/reference/argilla_sdk/records/suggestions.md

```diff
@@ -16,8 +16,8 @@ dataset.records.add(
         {
             "text": "Hello World, how are you?",
             "label": "negative",
-            "label.response": "negative",
-            "label.score": 0.9,
+            "label.suggestion.response": "negative",
```

Comment:
Sorry. I saw this and not sure if `suggestion.value` instead of `suggestion.response`. I think we're already using .value on exports?

Reply:
You're right. I missed that there was a response. I've changed the example to make it clearer how it works.